Makefile

```diff
@@ -45,19 +44,6 @@ clean: ## Cleans the underlying build.
 
 	rm -rf protoc-gen-connect-kotlin/src/test/java/
 
-.PHONY: crosstestserverrun
-crosstestserverrun: crosstestserverstop ## Run the server for cross tests.
-	docker run --rm --name serverconnect -p 8080:8080 -p 8081:8081 -d \
-		bufbuild/connect-crosstest:$(CROSSTEST_VERSION) \
-		/usr/local/bin/serverconnect --h1port "8080" --h2port "8081" --cert "cert/localhost.crt" --key "cert/localhost.key"
-	docker run --rm --name servergrpc -p 8083:8083 -d \
```

Comment:
NOTE: I never saw anything connect to anything other than the h2port of the connect server. If we want to extend the tests to hit the gRPC server we should do that.